extensions/amp-bind/0.1/test/test-bind-impl.js

```diff
@@ -135,6 +135,43 @@ describes.realWin('amp-bind', {
     });
   });
 
+  it('should dynamically detect new bindings added under certain tags', () => {
+    const doc = env.win.document;
+    const template = doc.createElement('template');
+    doc.getElementById('parent').appendChild(template);
+    return onBindReady().then(() => {
+      expect(bind.boundElements_.length).to.equal(0);
+      const textElement = createElementWithBinding('[onePlusOne]="1+1"');
+      template.appendChild(textElement);
+      return bind.waitForAllMutationsForTesting();
+    }).then(() => {
+      expect(bind.boundElements_.length).to.equal(1);
+    });
+  });
+
+  //TODO(kmh287): Move to a different test file?
```

Comment:
Putting them under a describe makes it easier to read. Done. I don't think there's enough setup that both need to justify a beforeEach